--- policycoreutils-1.6/scripts/checkcon.orig	Thu Jan 29 19:51:00 2004
+++ policycoreutils-1.6/scripts/checkcon	Tue Feb 24 14:30:39 2004
@@ -3,7 +3,7 @@
     echo "usage: $0 path ..."
     exit 1
 fi
-make -C /etc/security/selinux/src/policy file_contexts/file_contexts 2&>1 > /dev/null
+make -C /etc/security/selinux/src/policy file_contexts/file_contexts 2>&1 > /dev/null
 relabels=`echo $@ | tr " " "\n" | setfiles -q -v -n -s /etc/security/selinux/src/policy/file_contexts/file_contexts 2> /dev/null | grep relabeling`
 if [ -z "$relabels" ]; then
 exit 0
--- policycoreutils-1.6/scripts/restorecon.orig	Thu Jan 29 19:51:00 2004
+++ policycoreutils-1.6/scripts/restorecon	Tue Feb 24 14:30:47 2004
@@ -3,5 +3,5 @@
     echo "usage: $0 path ..."
     exit 1
 fi
-make -C /etc/security/selinux/src/policy file_contexts/file_contexts 2&>1 > /dev/null
+make -C /etc/security/selinux/src/policy file_contexts/file_contexts 2>&1 > /dev/null
 echo $@ | tr " " "\n" | /usr/sbin/setfiles -v -s /etc/security/selinux/src/policy/file_contexts/file_contexts 2> /dev/null 
